include/lsst/afw/image/Calib.h

```diff
@@ -51,6 +52,29 @@ namespace cameraGeom {
     class Detector;
 }
 namespace image {
+
+static double const JanskysPerABFlux = 3631.0;
+
+/// Compute AB magnitude from flux in Janskys
+inline double abMagFromFlux(double flux) {
+    return -2.5*std::log10(flux/JanskysPerABFlux);
+}
+
+/// Compute AB magnitude error from flux and flux error in Janskys
+inline double abMagErrFromFluxErr(double fluxErr, double flux) {
+    return std::abs(fluxErr/(-0.4*flux*std::log(10)));
+}
```

Comment:
I don't personally care about this, but I still feel obligated to point out that our name standards would likely prefer these to start with a verb (probably `compute`, unless we want to go with `get` for consistency with `Calib`, except that `Calib`'s should probably be `compute` as well, according to the standards)...

Also, to be especially troublesome, I'll point out that our standards also *require* the name of the methods below to use "Ab" instead of "AB", which I find just horrible (but then, I'm not a big fan of Psf, Wcs, or Lsst either).